packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts

```diff
@@ -0,0 +1,35 @@
+import { OptionsWithUri } from 'request';
+import {
+	IExecuteFunctions,
+	IExecuteSingleFunctions,
+	ILoadOptionsFunctions,
+} from 'n8n-core';
+import { IDataObject, NodeApiError, NodeOperationError, } from 'n8n-workflow';
+
+export async function uptimeRobotApiRequest(this: IExecuteFunctions | IExecuteSingleFunctions | ILoadOptionsFunctions, method: string, resource: string, body: any = {}, qs: IDataObject = {}, uri?: string, option: IDataObject = {}): Promise<any> { // tslint:disable-line:no-any
```

Comment:
Suggestion: Every async function returns a `Promise` and an unspecified return type implies `any`, so you can remove `Promise<any>` while preserving type information, including removal of the TSLint exception.